recipes/freeglut/all/test_package/conanfile.py

```diff
@@ -14,7 +14,6 @@ def layout(self):
 
     def requirements(self):
         self.requires(self.tested_reference_str)
-        self.requires("opengl/system")
```

Comment:
Please explain. How does it not require OpenGL at runtime?

Reply:
I believe it still requires it, but the OpenGL dependency is pulled in transitively from the actual FreeGLUT package. In the case of `libglvnd`, I would assume that nothing actually needs dispatched to a graphics driver in the test package, which seems possible since it appears to only be registering some FreeGLUT error handling functions and printing the FreeGLUT version.